app/views/active_admin/resource/_page_block_quote_component_form.html.arb

```diff
@@ -0,0 +1,37 @@
+placeholder ||= 'NEW_PAGE_COMPONENT_RECORD'
+component ||= nil
+html = Arbre::Context.new do
+  li "", id: "PageBlockQuoteComponent_poly_#{placeholder}", class: "input polyform component-#{placeholder}" do
+    fieldset class: "inputs" do
+      legend do
+        span 'Block Quote'
+      end
+
+      if component
+        input type: 'hidden', value: component.id, name: "page[page_components_attributes][#{placeholder}][component_attributes][id]"
+      end
+
+    	ol do
+        li class: 'string input optional stringish', id: "page_page_components_attributes_news_#{placeholder}_component_attributes_text_input" do
```

Comment:
```suggestion
        li class: 'string input required stringish', id: "page_page_components_attributes_news_#{placeholder}_component_attributes_text_input" do
```

Can you add a `required` class on these two fields? I have an upcoming PR to apply some styling to required fields